File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        
        int left = 0;
        int right = 1;

        while(right  < nums.size()){
            if(nums[right - 1] == nums[right]){
                right++;
            }
            else{
                nums[left] = nums[right - 1];
                left++;
                right++;
            }
        }
        nums[left] = nums[right - 1];
        left++;
        return left;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        left = 0
        right = 1

        while right < len(nums):
            if nums[right - 1] == nums[right]:
                right += 1
            else:
                nums[left] = nums[right - 1]
                left += 1
                right += 1

           

        nums[left] = nums[right - 1]
        left += 1

        return left

       
        